pg/source-create/pluto.sql

```diff
@@ -0,0 +1,37 @@
+DROP TABLE IF EXISTS 
+	source_land_use,
+	source_pluto,
+	source_zoning_district,
+	source_zoning_district_class
+	CASCADE;
+
+CREATE TABLE IF NOT EXISTS source_land_use (
```

Comment:
Why do land_use and zoning_district have source tables created here in pluto.sql when borough is in its own file if they're all directly loading from static csvs?

Reply:
It's less about the literal csvs and more about the "domain". Land uses and zoning districts are part of the "pluto" domain. Boroughs are closer to the "admin" domain. But, because they have dependents across admin and pluto domains, they are their own thing. 

[diagram of domains](https://raw.githubusercontent.com/NYCPlanning/ae-data-flow/832afe69a29290f215a7d1ec05f20e480e0c7f28/documentation/diagrams/domains.drawio.png) (diagram on the left)

[short blurb on domains](https://github.com/NYCPlanning/ae-data-flow/pull/57/files/832afe69a29290f215a7d1ec05f20e480e0c7f28#diff-164368f345e1ac5e655c169f62f221451b28a015f8a11b47c76793b099e12bbaR108)